.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+
+<your comments for this PR go here>
+
+Have you read README.md lately? If not, do that first.
+
+---
+
+- [ ] Review code comments; remove done TODOs, create stories for remaining TODOs
+- [ ] Optionally link to the Jira Ticket for the convenience of the reviewer
+- [ ] Run all tests, both **unit** and **integration** tests
+- [ ] Preface the PR title with the Jira Ticket Number and security risk, i.e., "[RW-123][risk=no] My New Feature"
```

Comment:
My suggestion was going to be: (to even further minimize the no risk case.)
[]
[risk = low]
[risk = moderate]
[risk = severe]